.env

```diff
@@ -0,0 +1,11 @@
+# Make a copy of this file and update the variables.
```

Comment:
Should this file be called `.env-template` or something, and `.env` be added to `.gitignore` to prevent that someone accidentally pushes changes to GitHub?

Reply:
Good point. I'll add it to `.gitignore` and rename the example.
I think they can be defined in the same file. I was thinking to have only one configuration file for the whole project, which could be set using environmental variables, through the `.env` file or explicitly by the user, or through a configuration in `json`, `toml` or `yaml` format.